frontend/package.json

```diff
@@ -35,5 +35,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
These should be added to `dependencies`.

Reply:
That is made that way by create-react-app. So I don't think changing that is a good idea